bridge/jira/jira.go

```diff
@@ -19,13 +17,14 @@ const (
 	metaKeyJiraDerivedId  = "jira-derived-id"
 	metaKeyJiraKey        = "jira-key"
 	metaKeyJiraUser       = "jira-user"
-	metaKeyJiraProject    = "jira-project"
+	metaKeyJiraBridge     = "jira-bridge"
```

Comment:
not sure, I've planned to use the bridge as the selector. ie: the bridge that import should do the export. I think it's a more strict this way. This is because I'm replacing the project constraint with the filter if defined.

Reply:
We can have a back-ward mechanism, will preserve the project meta key and use it if there is no jira bridge key defined. The jira bridge meta key will be only used when the user will have provided a filter overriding the project filter.